module/platform/OpenCR/HardwareIO/src/HardwareIO.cpp

```diff
@@ -169,10 +107,68 @@ namespace module::platform::OpenCR {
             // The model watchdog is started, which has a longer time than the packet watchdog
             // The packet watchdog is disabled until we start the main loop
             model_watchdog.enable();
-            packet_watchdog.disable();
 
             // The startup function sets up the subcontroller state
             startup();
+
+            packet_watchdog =
```

Comment:
Would it make sense to move the model watchdog creation here, then create the packet watchdog when we get model information so that no overlap can ever occur